lib/pymedphys/_mosaiq/connect.py

```diff
@@ -184,24 +195,28 @@ def _connect_with_credential_then_prompt_if_fail(
 
     """
     server, port = _separate_server_port_string(sql_server_and_port)
+    storage_name = f"{KEYRING_SCOPE}_{sql_server_and_port}/{database}"
```

Comment:
form the keyring storage name from: KEYRING_SCOPE, server + port, and database name